apps/prairielearn/src/lib/externalGrader.js

```diff
@@ -79,23 +94,23 @@ module.exports._beginGradingJob = function (grading_job, submission, variant, qu
 
   logger.verbose(`Submitting external grading job ${grading_job.id}.`);
 
-  const gradeRequest = module.exports.grader.handleGradingRequest(
+  const gradeRequest = grader.handleGradingRequest(
     grading_job,
     submission,
     variant,
     question,
     course,
   );
   gradeRequest.on('submit', () => {
-    updateJobSubmissionTime(grading_job.id, (err) => {
-      if (ERR(err, (err) => logger.error('Error updating job submission time', err))) return;
+    updateJobSubmissionTime(grading_job.id).catch((err) => {
+      logger.error('Error updating job submission time', err);
```

Comment:
Nit: as long as you're touching this code, consider adding `Sentry.captureException` here.

Reply:
Just so I don't add too many of these: which exceptions should be added to Sentry? Only those that are unlikely to happen in normal operations? For example, an error processing the grading results is typically caused by bugs in instructor grader code, should it trigger sentry?

Reply:
Done in 3013ff46d3f96a9cb0568e89d8d6729b752b0614. I have not added it to errors caused by `processGradingResult`, let me know if I should add them there.

Reply:
Right, we only want to capture exceptions that are caused by our own application code (that is, errors that we could actually fix).